editor/src/components/editor/actions/actions.tsx

```diff
@@ -4112,7 +4112,12 @@ export const UPDATE_FNS = {
       withPackageJson,
       action.parentPath,
       newFileName,
-      codeFile(templateFile.fileContents.code, RevisionsState.CodeAhead),
+      codeFile(
+        templateFile.fileContents.code,
+        templateFile.fileContents.code,
```

Comment:
This should be getting `null` for `lastSavedContents`, otherwise it is considered to be unsaved

Reply:
added it in https://github.com/concrete-utopia/utopia/pull/5738/commits/aee4b835ad1c7ab7da374b3fa6ea73033895a8a2

Reply:
> This should be getting `null` for `lastSavedContents`, otherwise it is considered to be unsaved

Makes me think we should create functions that set these values appropriately like a `newCodeFile` and `editedCodeFile` or whatever.